src/Controllers/SubmitControllerPost.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Stats\Controllers;
+
+use Joomla\Controller\AbstractController;
+use Stats\Models\StatsModel;
+
+/**
+ * Controller for processing submitted statistics data.
+ *
+ * @method         \Stats\Application  getApplication()  Get the application object.
+ * @property-read  \Stats\Application  $app              Application object
+ *
+ * @since          1.0
+ */
+class SubmitControllerPost extends AbstractController
+{
+	/**
+	 * Statistics model object.
+	 *
+	 * @var    StatsModel
+	 * @since  1.0
+	 */
+	private $model;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param   StatsModel  $model  Statistics model object.
+	 *
+	 * @since   1.0
+	 */
+	public function __construct(StatsModel $model)
+	{
+		$this->model = $model;
+	}
+
+	/**
+	 * Execute the controller.
+	 *
+	 * @return  boolean
+	 *
+	 * @since   1.0
+	 * @throws  \RuntimeException
+	 */
+	public function execute()
+	{
+		$input = $this->getInput();
+
+		$data = [
+			'php_version' => $input->getRaw('php_version'),
+			'db_version'  => $input->getRaw('db_version'),
+			'cms_version' => $input->getRaw('cms_version')
+		];
+
+		// Filter the submitted version data.
+		$data = array_map(
+			function ($value)
```

Comment:
It's an argument for the anonymous function, its behavior is the same as arguments for "normal" methods (like the `__construct` method above).  So whatever value is injected into the method as PHP does its thing gets assigned to that `$value` variable.